openvoxserver/Containerfile

```diff
@@ -76,28 +78,27 @@ COPY docker-entrypoint.d /docker-entrypoint.d
 COPY --from=builder /var/lib/gems/ /var/lib/gems/
 COPY --from=builder /usr/local/bin/r10k /usr/local/bin/
 
-ADD https://s3.osuosl.org/puppet-artifacts/openvox-server/${OPENVOXSERVER_VERSION}/openvox-server_${OPENVOXSERVER_VERSION}-1%2Bubuntu24.04_all.deb /
-ADD https://s3.osuosl.org/puppet-artifacts/openvox-agent/${OPENVOXAGENT_VERSION}/openvox-agent_${OPENVOXAGENT_VERSION}-1%2Bubuntu24.04_${TARGETARCH}.deb /
-ADD https://s3.osuosl.org/puppet-artifacts/openvoxdb/${OPENVOXDB_VERSION}/openvoxdb-termini_${OPENVOXDB_VERSION}-1%2Bubuntu24.04_all.deb /
+ADD https://apt.overlookinfratech.com/openvox${OPENVOX_RELEASE}-release-ubuntu${UBUNTU_VERSION}.deb /
+RUN apt-get update && \
+    apt-get install -y ca-certificates && \
+    dpkg -i /openvox${OPENVOX_RELEASE}-release-ubuntu${UBUNTU_VERSION}.deb && \
```

Comment:
Apt can install local files just fine

Reply:
oh okay didnt know... will try that. is it like in yum?

Reply:
Kind of, I'd say yum does it a little better because you can also point it to http(s) URLs. With apt you need to get it somehow (like wget) and the use `apt install /path/to/file.deb`. You can't use a relative path so you must use `./file.deb` if it's in the current directory.